### DISCUSSION TITLE
Nova custom field not changing / not compiling

### DISCUSSION THREAD

**Discussion by baszach** [laravel/nova-issues#4620] [Q&A]
Anyone else came across an issue where your Nova field just won't accept the changes in the browser (even though npm run watch is running)?

I removed a line from a Vue component (for a Nova field) and tried many things but in the browser it still displays the old version:

- removed node_modules + dist in field directory and 
- removed node_modules + vendor in project root and
- restarted my docker container cleared all volumes and restarted
- cleared browser cache

Could there be some other kind of cache somewhere? It has happened to me a few times that changes on my custom Nova components appeared after several minutes or hours...

**Comment by ZoreAnkit** [comment]
You can try following steps:

`Disable browser cache` -
 Inspect browser -> Network tab -> Click Disable cache -> refresh browser

`In Folder` - 
Navigate to component -> Delete node_modules -> Delete package-lock.json -> npm install -> npm run watch(to check changes)

**Reply by baszach** [reply]
Thanks. I tried that. Didn't work sadly.

But I got this reply https://github.com/laravel/nova-issues/discussions/4549#discussioncomment-3362617 from a Nova maintainer. Maybe this will be helpful.
